library/koji_tag.py

```diff
@@ -517,31 +507,31 @@ def ensure_tag(session, name, check_mode, inheritance, external_repos,
                 session.editTag2(name, **edits)
 
     # Ensure inheritance rules are all set.
-    if compound_parameter_present('inheritance', inheritance, list):
+    if inheritance:
```

Comment:
Looking at the integration test failure, it seems we cannot clear out a value if the user wants to do that.

In #155, I said to use "if variable:", and I think we actually want "`if variable is not None`" here.

Reply:
(And I'll need to watch out for this when we fix #123)

Reply:
I just noticed that myself, I've updated the commit.

Reply:
I should have waited! Thank you.